packages/launchdarkly-adapter/modules/adapter/adapter.ts

```diff
@@ -231,6 +231,10 @@ class LaunchDarklyAdapter implements LaunchDarklyAdapterInterface {
     return adapterState.isReady;
   }
 
+  getClient(): LDClient | undefined {
```

Comment:
Would you prefer the client to be exposed fully or just the track function?

Reply:
Well - I figured my more isolated usecase of just needing the track function may not be what others may need in the future - so felt returning the entire client would save us the headache of needing to do a follow-up PR.

Reply:
Yup. However, at the cost that in theory and breaking change to the client's release cadence, we don't control, being a breaking change to this package. But I don't mind syncing those by now.,

Reply:
Thanks for getting this in! Really looking forward to having this released. Is that something I can submit a PR for as well by chance?

Reply:
Let me see if canary releases still with with GitHub Actions. I saw some misconfig from my end lately. Otherwise I can publish it later.